specification/TileFormats/PointCloud/README.md

```diff
@@ -171,6 +171,32 @@ This is useful for per-object picking and storing application-specific metadata
 The `BATCH_ID` semantic may have a `componentType` of `UNSIGNED_BYTE`, `UNSIGNED_SHORT`, or `UNSIGNED_INT`. When `componentType` is not present, `UNSIGNED_SHORT` is used.
 The global semantic `BATCH_LENGTH` defines the number of unique `batchId` values, similar to the `batchLength` field in the [Batched 3D Model](../Batched3DModel/README.md) header.
 
+### Draco
+
+Per-point semantics such as position, color, normal, and batch id may may be compressed with [Draco](https://google.github.io/draco/). Draco binary data is stored in the Feature Table binary and a list of the compressed per-point semantics is stored in the Feature Table JSON.
+
+The `DRACO` object in the Feature Table JSON header contains three properties:
+   * `byteOffset` specifies a zero-based offset relative to the start of the Feature Table binary body.
+   * `byteLength` specifies the length of the Draco binary data.
+   * `semantics` is an array containing the per-point semantics contained in the Draco binary data. Allowed semantics are `"POSITION"`, `"RGBA"`, `"RGB"`, `"NORMAL"`, and `"BATCH_ID"`.
+
+Point attribute data shall be encoded and decoded using the parameters in the following table.
```

Comment:
Should we specify that they must be encoded/decoded with the draco tools as well (and link to them)?